.changeset/rare-ears-rhyme.md

```diff
@@ -0,0 +1,6 @@
+---
+"@launchpad-ui/icons": minor
+"@launchpad-ui/core": minor
```

Comment:
```suggestion
```
Changesets will automatically update `core` and nothing breaks from a core's usage perspective

Reply:
ahhhh, gotcha. Last time I read the CONTRIBUTING docs, it had the line about _always_ including core. but I see that's since been removed! 

https://github.com/launchdarkly/launchpad-ui/commit/bdd6f5c24d9d4f1237e3269cb917b5b935bdd89f